src/puppetlabs/puppetdb/query_eng/engine.clj

```diff
@@ -667,13 +667,31 @@
         [:null name]
         [field name]))))
 
+(defn merge-function-options
+  "Optionally merge call and grouping into an existing query map.
+   Alias function calls with escape-quoted function name for hsqldb compatability."
+  [m call grouping]
+
+  (if-not call
+    m
+    (let [fn-call (apply hcore/call call)
+          fn-alias (format "\"%s\"" (name (first call)))]
+      (-> m
+          (update-in [:select] conj [fn-call fn-alias])
+          (#(if grouping (assoc % :group-by (map keyword grouping)) %))))))
+
 (defn honeysql-from-query
-  [{:keys [selection projections paging-options] :as query}]
+  [{:keys [projected-fields group-by call selection projections
```

Comment:
Is there any way to make call a little more general, that we might more easily support multiple functions in a query at a later date?